tests/cypress/e2e/components/media.cy.ts

```diff
@@ -67,15 +67,15 @@ const mediaType = [
     tagRelated: 'model-viewer',
     type: '3d',
   },
-  {
-    url: '/bsx/gallery/2444376227-3',
-    title: '3D Ball',
-    description: 'Smash zone adverse',
-    collection: 'Ping Pong Ball',
-    creator: 'bXhWeV...2sDcbW',
-    tagRelated: 'model-viewer',
-    type: '3d',
-  },
+  // {
+  //   url: '/bsx/gallery/2444376227-3',
+  //   title: '3D Ball',
+  //   description: 'Smash zone adverse',
+  //   collection: 'Ping Pong Ball',
+  //   creator: 'bXhWeV...2sDcbW',
+  //   tagRelated: 'model-viewer',
+  //   type: '3d',
+  // },
```

Comment:
☹️ is it breaking?

Reply:
seems this PR https://github.com/kodadot/nft-gallery/pull/3839 sometimes not really solved the issue, run e2e in local seems fine with that enabled

Reply:
oh, the test was correct, broken on bsx. checking

Reply:
sorry, turns out not working on bsx after update from main branch. will check later, currently afk 🙏🏻